services/QuillLMS/client/app/bundles/Teacher/containers/EditOrCreateSubscription.jsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import { SingleDatePicker } from 'react-dates'
 import _ from 'lodash';
-import request from 'request';
 import { DataTable } from '../../Shared/index'
 import ItemDropdown from '../components/general_components/dropdown_selectors/item_dropdown.jsx';
 import getAuthToken from '../components/modules/get_auth_token';
```

Comment:
I think `import getAuthToken` and authenticity_token further down can be removed.

Reply:
There's also one more of the `import getAuthToken` and corresponding authenticity_token that I think can be removed  in:  https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/bundles/Teacher/containers/TeacherAccount.jsx#L10 but for some reason I'm not showing that in this PR.